tcms/core/contrib/auth/views.py

```diff
@@ -36,7 +35,7 @@ def register(request, template_name='registration/registration_form.html'):
         form = RegistrationForm(data=request.POST, files=request.FILES)
         if form.is_valid():
             new_user = form.save()
-            ak = form.set_active_key()
+            active_key = form.set_active_key()
```

Comment:
This is activation_key and both the form method and the send_mail parameter bame needs to reflect that